cmd/cli/commands/worker.go

```diff
@@ -100,10 +107,15 @@ var workerCurrentCmd = &cobra.Command{
 		if len(cfg.WorkerAddr) == 0 {
 			addr := crypto.PubkeyToAddress(getDefaultKeyOrDie().PublicKey)
 			cmd.Printf("current worker is not set, using cli's addr %s\n", addr.Hex())
-		} else if !common.IsHexAddress(cfg.WorkerAddr) {
-			cmd.Printf("current worker(%s) is invalid\n", cfg.WorkerAddr)
 		} else {
-			cmd.Printf("current worker is %s\n", cfg.WorkerAddr)
+			addr, err := auth.NewAddr(cfg.WorkerAddr)
+			if err != nil {
+				cmd.Printf("current worker(%s) is invalid\n", cfg.WorkerAddr)
+			} else if _, err := addr.ETH(); err != nil {
```

Comment:
Please be consistent. In this PR there are two almost equal checks. Rewrite them to be at least looking similar.